src/dal_select2/static/autocomplete_light/select2.js

```diff
@@ -6,7 +6,8 @@
         $(this).find('option[value=""]').remove();
 
         $(this).select2({
-            tags: element.attr('data-tags'),
+            tags: element.attr('data-tags') ? true : false,
```

Comment:
Select2 _should_ [automatically parse `data-tags`](https://select2.github.io/options.html#data-attributes) and enable tagging.